addons/sale_timesheet/views/hr_timesheet_views.xml

```diff
@@ -173,58 +173,83 @@
     <!--
         Reporting
     -->
+    <record id="timesheets_analysis_report_pivot_inherit" model="ir.ui.view">
+        <field name="name">timesheets.analysis.report.pivot</field>
+        <field name="model">timesheets.analysis.report</field>
+        <field name="inherit_id" ref="hr_timesheet.timesheets_analysis_report_pivot_employee"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='unit_amount']" position="after">
+                <field name="billable_time" widget="timesheet_uom"/>
+                <field name="non_billable_time" widget="timesheet_uom"/>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="timesheets_analysis_report_graph_inherit" model="ir.ui.view">
+        <field name="name">timesheets.analysis.report.graph</field>
+        <field name="model">timesheets.analysis.report</field>
+        <field name="inherit_id" ref="hr_timesheet.timesheets_analysis_report_pivot_employee"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='unit_amount']" position="after">
+                <field name="billable_time" widget="timesheet_uom"/>
+                <field name="non_billable_time" widget="timesheet_uom"/>
+            </xpath>
+        </field>
+    </record>
```

Comment:
Since it is now a report, we could move the views and actions definition of this report to the report folder of this module. 😇